deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -629,6 +620,10 @@ private void ingestWork(ContentContainerObject parent, Resource parentResc, Reso
 
                 overrideModifiedTimestamp(obj, childResc);
 
+                // Add counts for files deposited for the work
```

Comment:
Wrap these two calls in an `if (filesIngested > 0)` block in case there were no files.